src/screens/Connections/Connections.tsx

```diff
@@ -22,7 +22,11 @@ import EmptyState from './components/EmptyState/EmptyState';
 import Switch from './components/Switch/Switch';
 import {flights} from '../../data';
 
-const Connections = () => {
+type Props = {
+  navigation: any;
```

Comment:
Avoid using any as type. Here you can specify the NavigationProp as type, or you can use the useNavigation hook for getting the navigation object.